dts/bindings/usb/usb.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2018, I-SENSE group of ICCS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: USB Base Structure
+id: USB
+version: 0.1
+
+description: >
+    This binding gives the base structures for all USB devices
+
+inherits:
+    !include usb-ep.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+
+    clock-frequency :
```

Comment:
What is the clock-frequency property used for?

Reply:
It is not used. I will remove it.